rust/matches/src/utils.rs

```diff
@@ -315,7 +315,7 @@ pub fn verify(proof: &Vec<[u8; 32]>, root: &[u8; 32], leaf: [u8; 32]) -> bool {
 pub fn is_part_of_namespace<'a>(artifact: &AccountInfo<'a>, namespace: &Pubkey) -> bool {
     let data = artifact.data.borrow_mut();
     let number = u32::from_le_bytes(*array_ref![data, 9, 4]) as usize;
-    let offset = 13 as usize;
+    let offset = 13_usize;
```

Comment:
I've never seen this before. How does that work?

Reply:
apparently this is with the new anchor 0.25.0 linting